runtime/calamari/src/lib.rs

```diff
@@ -369,7 +369,7 @@ impl frame_system::Config for Runtime {
     type OnNewAccount = ();
     type OnKilledAccount = ();
     type AccountData = pallet_balances::AccountData<Balance>;
-    type SystemWeightInfo = weights::frame_system::SubstrateWeight<Runtime>;
+    type SystemWeightInfo = ();
```

Comment:
seems odd

Reply:
we're having problem on benchmarking `set_code`(the error `ValidationDataNotAvailable`), `cumulus-pallet-parachain-system` brought the problem here though `runtime-benchmarks` has been enabled for this this pallet.

Reply:
And use default weights should be fine, some parachain like acala who is using default weights